content/pt-br/style-guide/_index.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Guia de Estilo
+toc_hide: true
+menu:
+  main:
+    weight: 10
+    pre: <i class='fas fa-ruler-horizontal'></i>
+---
+
+Este guia de estilo ajudará você a entender o público alvo do glossário, estrutura das definições, nível de detalhamento necessário e como manter um estilo consistente.
+
+O glossário Cloud Native segue o [guia de estilo padrão da CNCF](https://github.com/cncf/foundation/blob/master/style-guide.md). Além disso, segue as seguintes regras:
+
+1. Use uma linguagem simples e acessível, evitando jargões técnicos e *buzzwords*
+2. Evite usar [linguagem coloquial](https://pt.wikipedia.org/wiki/Linguagem_coloquial)
+3. Use uma [linguagem literal e concreta](http://guidetogrammar.org/grammar/composition/abstract.htm)
+4. Evite usar [contrações](https://en.wikipedia.org/wiki/Contraction_(grammar))
+5. Use a [voz passiva]((https://www.ef.com/ca/english-resources/english-grammar/passive-voice/)) com moderação
+6. Busque usar [declarações de forma positiva](https://examples.yourdictionary.com/positive-sentence-examples.html) 
+7. Em citações, não use [pontos de exclamação fora das aspas](https://www.grammarly.com/blog/exclamation-mark/)
+8. Evita usar exageros
+9. Evite usar repetições
+10. Seja conciso
+
+## Público alvo
+
+O glossário foi escrito para um público alvo técnico e não técnico. Certifique-se de que as definições estejam explicadas em termos simples que não presumam conhecimento técnico prévio. 
+
+## Modelo para as definições
+
+Cada definição do glossário é escrita em um arquivo markdown, baseada no seguinte modelo:
+
+```md
+---
+title: 
+status: 
+category: 
+---
+
+## What it is
```

Comment:
I guess this is not correct?!

Reply:
@iamNoah1 I think it is correct :) 

`code` applied from line 40 ~ line 51 

PTAL: https://deploy-preview-675--cncfglossary.netlify.app/pt-br/style-guide/#modelo-para-as-defini%C3%A7%C3%B5es

Reply:
Hi @iamNoah1! As Seokho said, this is a code block to show the original structure of a definition file, On the [Definition topic](https://deploy-preview-675--cncfglossary.netlify.app/pt-br/style-guide/#subcabe%C3%A7alhos) I've translated the subheading to Portuguese :)

Reply:
No worries, thanks for your detailed review @iamNoah1!